Gemfile

```diff
@@ -64,6 +64,7 @@ gem 'wkhtmltopdf-binary'
 gem 'foreigner'
 gem 'immigrant'
 gem 'roo', '~> 2.7.0'
+gem 'roo-xls'
```

Comment:
Can do. There's only one version and they havn't changed it in 2 years so I thought it wasn't too important. It's for legacy excel support, which probably won't change...